packages/kotti-ui/source/kotti-card/KtCard.vue

```diff
@@ -53,21 +47,19 @@ export default defineComponent<KottiCard.PropsInternal>({
 	flex-direction: column;
 	word-break: break-all;
```

Comment:
Apparently I added this with my refactor, at least it is not present in the [original css](https://github.com/3YOURMIND/kotti/pull/575/files#diff-0a1dd74771df923f355b00ac7962ef75f54ff5f4567989ff40d0b854417c47c6).

This forces very ugly line breaks (see screenshot in my other comment, can also be seen in [current documentation](https://3yourmind.github.io/kotti/usage/components/card)). I don't know why I did that. I'd say remove or at least use `break-word` instead of `break-all`

Reply:
can do break-word, no worries